cardazim/connection.py

```diff
@@ -0,0 +1,54 @@
+import socket
+import struct
+
+
+class Connection:
+    def __init__(self, s):
+        self.s = s
+
+    def __repr__(self):
+        return f"<Connection from {self.s.getsockname()[0]}:{self.s.getsockname()[1]} to {self.s.getpeername()[0]}:{self.s.getpeername()[1]}>"
```

Comment:
What do you think of something like this? You don't have to change it if you don't like it.
```suggestion
        from_ip, from_port = self.s.getsockname()
        to_ip, to_port = self.s.getpeername()
        return f"<Connection from {from_ip}:{from_port} to {to_ip}:{to_port}>"
```